rfc/ipni-w3c.md

```diff
@@ -2,53 +2,54 @@
 
-This RFC describes how IPNI indexing can be used to lookup data locations and content claims for any CID within a CAR stored in w3c.
+This RFC describes how IPNI indexing can be used to lookup data locations and content claims for any piece of data, identified by a multihash, stored in w3c.
 
 ## Abstract
 
-A user stores content as opaque blobs containing series of blocks addressed by multihashes. Currently, these blocks are collected into a CAR file for storage as one body of data.
-
-The user creates an external CAR index [CARx-index](https://github.com/web3-storage/RFC/pull/9/files) that contains tuples of (multihash, offset, length) of any items the user intends to make individually retrievable from the car.
+A user stores content as opaque blobs containing series of blocks addressed by multihashes. Currently, these blocks are collected into a CAR file for storage as one body of data. This will be generalized into a blob of data that does not require a CAR file.
 
-With an [inclusion claim](https://hackmd.io/@gozala/content-claims?utm_source=preview-mode&utm_medium=rec#Inclusion-Claims), the user who created the CAR asserts that CAR contains a given set of multihashes, transitively, via the CARx-index.
+The user creates an [inclusion claim] that contains the multihash of the data blob and an array of tuples of (multihash, offset, length) of any items the user intends to make individually retrievable from their data blob. This inclusion claim serves as an external index into the user's data that can be stored and retrieved separately from the data. This inclusion claim is also the user's assertion that their data blob contains the data represented by a given set of multihashes.
 
-After storing the CAR and CARx-index, w3s creates a [location claim](https://hackmd.io/@gozala/content-claims?utm_source=preview-mode&utm_medium=rec#Location-Claims) for the CAR file and the CARx-index. Each location claim maps an item's CID to a URL where the item can be retrieved.
+After storing the data blob and the inclusion claim, w3s creates a [location claim](https://hackmd.io/@gozala/content-claims?utm_source=preview-mode&utm_medium=rec#Location-Claims) for the blob and the inclusion claim (serialized to bytes). Each location claim maps an item's CID to a URL where the item can be retrieved.
 
-w3s creates a Content Claims Bucket. This is a container, stored on w3s network, that holds all content claims associated with the stored data. The location claims and inclusion claim are written to the claims bucket. The claims bucket is then stored.
+w3s creates a Content Claims Bundle. This is a container, stored on the w3s network, that holds all content claims associated with the stored data. The location claims and inclusion claim are written to the claims bundle. The claims bundle is then stored in the user's storage space.
```

Comment:
This is better, thank you. You are correct that the user creates the inclusion claim and the service creates the location claim.

I think either the service or the user could create the claims bundle. I was thinking the service would create the bundle, since otherwise it would need to verify its presence before generating an IPNI advertisement, and that would require another message from the user to tell the service that the bundle had been written.